graph/src/categorisation.c

```diff
@@ -0,0 +1,80 @@
+#include "depmap.h"
+#include "categorisation.h"
+#include <stdlib.h>
+
+typedef struct {
+    bool ext_in;
+    bool ext_out;
+} rage_Flippy;
```

Comment:
A better name, perhaps? Something like `rage_Externalities`, `externs` for short reference?